common/source/docs/common-serial-led-neopixel.rst

```diff
@@ -5,11 +5,15 @@ NeoPixel
 
 .. image:: ../../../images/neopixel-string.jpeg
 
-A "NeoPixel" style (WS2812B compatible) RGB LED can be attached to any PWM output by setting its SERVOx_FUNCTION to one of the ``NeoPixelx`` output functions and setting :ref:`NTF_LED_TYPES<NTF_LED_TYPES>` parameter to ``NeoPixel``. Multiple ``NeoPixelx`` output functions are provided for connecting multiple strings (up to 4).
+Up to four "NeoPixel" style (WS2812B compatible) RGB LEDs can be used to display the same LED patterns as other RGB LEDs including those commonly installed on GPS/compass units
```

Comment:
This is a little misleading. Four outputs but each output can have multiple LEDs only limited by processing time.

Reply:
@andyp1per,

So how about, 

`Up to four strings of "NeoPixel" style (WS2812B compatible) RGB LEDs can be used to display the same LED patterns as other RGB LEDs including those commonly installed on GPS/compass units.  Each string can have multiple LEDs with the maximum number dependent upon the autopilot's CPU speed.  On a high speed autopilot with an H7 processor, up to 44 individual LEDs per string are known to work.`

If this doesn't sound good could you give a suggested wording?

Reply:
I think this is good

Reply:
ok update to what i've written above.